src/app/components/plugin-manager-component.js

```diff
@@ -203,11 +201,11 @@ class PluginManagerComponent extends ViewPlugin {
         <section>
           ${activeTile}
           <div class="list-group list-group-flush plugins-list-group">
-            ${actives.map(name => this.renderItem(name))}
+            ${actives.map(profile => this.renderItem(profile, isActive(profile.name)))}
```

Comment:
If you pass `profile` as parameter of `renderItem`, you don't need to pass the `isActive` params : 
```typescript
renderItem(profile) {
  const pluginIsActive = isActive(profile.name)
}
```
Like that, here you can do:
`actives.map(profile => this.renderItem(profile)`